doc/functions.sgm

```diff
@@ -149,6 +149,39 @@
             </example>
           </sect2>
 
+          <sect2 id="func.spoint.dist">
+            <title>
+              Point distance function
+            </title>
+            <para>
+              The function
+            </para>
+            <funcsynopsis>
+              <funcprototype>
+                <funcdef><function>spoint_dwithin</function></funcdef>
+                  <paramdef>spoint <parameter>p1</parameter></paramdef>
+                  <paramdef>spoint <parameter>p2</parameter></paramdef>
+                  <paramdef>float8 <parameter>radius</parameter></paramdef>
+              </funcprototype>
+            </funcsynopsis>
+            <para>
+              returns if points <parameter>p1</parameter> and <parameter>p2</parameter>
```

Comment:
Returns _what_? I think it should say what it returns.

Reply:
... and done. Pushed with two more doc tweaks, thanks for the review!

Reply:
This is better, but I'd still like to see "returns if" on line 168 changed to something like "returns a boolean value that signifies whether or not".

Reply:
Thanks, that's the wording I was missing. Updated as part of c9e410ff2.